### ISSUE TITLE
XSD links are dead

### ISSUE THREAD

**Issue by gamelaster** [propelorm/Propel2#1768]
`http://xsd.propelorm.org/1.6/database.xsd` is dead :(

**Comment by dereuromark** [comment]
@marcj Do you have access to the URL/Website?
How can we adjust and keep this up to date?

**Comment by marcj** [comment]
@dereuromark no, Veikko Mäkinen is the owner of the domain.

**Comment by dereuromark** [comment]
Do you have a contact or github name or alike we can reach out to?

**Comment by marcj** [comment]
You can try veikko@veikko.fi, this is the one also used in git commits here.

**Comment by gamelaster** [comment]
Also, please, note that sandbox is dead too.

**Comment by marcj** [comment]
Yes, sandbox should be removed from the website since it won't come back online.
